db/migrate/20170212055707_create_teams.rb

```diff
@@ -0,0 +1,16 @@
+class CreateTeams < ActiveRecord::Migration[5.0]
+  def change
+    create_table :teams do |t|
+      t.string :team_name
```

Comment:
aha, actually, unless someone has an objection I think I'll keep this name - just discovered that in the users table 'name' is actually the slack name, not the user's name.  I'll add first_name and last_name fields, and maybe rename 'name' to slack_name.
Or would it be better to rename users => slack_users and add a separate users table that can be 1:1 with slack_users (and other online services as we add them)?